File: Patterns/python/pattern5.py
# n = 4

# 1
# 1 2
# 1 2 3 
# 1 2 3 4

n = int(input())
row = 1
while row <= n:
    col = 1
    while col <= row:
        print(col, end="")
        col += 1
    row += 1
    print("")

File: Patterns/go/pattern5.go
// n = 4

// 1 2 3 4
// 1 2 3 4
// 1 2 3 4
// 1 2 3 4
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)
	
	row := 1
	for row <= n {
		col := 1
		for col <= n {
			fmt.Printf("%d", col)
			col++
		}
		row++
		fmt.Printf("\n")
	}
}